src/main/kotlin/no/nav/familie/baks/mottak/task/JournalhendelseBarnetrygdRutingTask.kt

```diff
@@ -54,10 +54,13 @@ class JournalhendelseBarnetrygdRutingTask(
 
     val log: Logger = LoggerFactory.getLogger(JournalhendelseBarnetrygdRutingTask::class.java)
     val sakssystemMarkeringCounter = mutableMapOf<String, Counter>()
+    val enheterSomIkkeSkalHaAutomatiskJournalføring = listOf("4863")
```

Comment:
Vi burde nok hatt en enum eller navngitt konstant for å forklare hva 4863 er her, keen på å legge det kjapt inn @UyQuangNguyen ?